ksql-rest-app/src/test/java/io/confluent/ksql/parser/ParserMatchers.java

```diff
@@ -146,7 +148,7 @@ public StatementTextMatcher(Matcher<? super String> textMatcher) {
 
     @Override
     protected String featureValueOf(final PreparedStatement<T> actual) {
-      return actual.getStatementText();
+      return actual.getMaskedStatementText();
```

Comment:
How come this one's masked even though the ones above are unmasked?

Reply:
They should all be unmasked, good catch!